internal/record/recording_https_proxy.go

```diff
@@ -98,7 +101,10 @@ func (r *RecordingHTTPSProxy) recordRequest(req *http.Request) (string, error) {
 		return "", err
 	}
 
-	recordedRequest.RedactHeaders(r.config.RedactRequestHeaders)
+	r.redactor.Headers(recordedRequest.Header)
```

Comment:
are line 104 and 106 duplicate?

Reply:
Good catch, the previous line here was actually redacting headers by key, fixed.